futures/mark_price.go

```diff
@@ -115,3 +115,64 @@ type FundingRate struct {
 	FundingTime int64  `json:"fundingTime"`
 	Time        int64  `json:"time"`
 }
+
+// GetLeverageBracketService get funding rate
+type GetLeverageBracketService struct {
+	c      *Client
+	symbol string
+}
+
+// Symbol set symbol
+func (s *GetLeverageBracketService) Symbol(symbol string) *GetLeverageBracketService {
+	s.symbol = symbol
+	return s
+}
+
+// Do send request
+func (s *GetLeverageBracketService) Do(ctx context.Context, opts ...RequestOption) (res []*LeverageBracket, err error) {
+	r := &request{
+		method:   "GET",
+		endpoint: "/fapi/v1/leverageBracket",
+		secType:  secTypeSigned,
+	}
+	r.setParam("symbol", s.symbol)
+	if s.symbol != "" {
+		r.setParam("symbol", s.symbol)
+	}
+	data, err := s.c.callAPI(ctx, r, opts...)
+	if err != nil {
+		return []*LeverageBracket{}, err
+	}
+
+	if s.symbol != "" {
```

Comment:
We can use `common.ToJSONList` to forcedly convert data to a List, please refer to https://github.com/adshao/go-binance/blob/7314d5890dc68c918ef88ff5f2b5267dc2e8b154/futures/ticker_service.go#L32